### DISCUSSION TITLE
Mobile Compatibility

### DISCUSSION THREAD

**Discussion by faytekin** [opcodesio/log-viewer#75] [Ideas]
It cannot be viewed from mobile devices. 
Can it be made mobile compatible?

Now there is an image like the one below. Log files are not shown.

![IMG_6AD055A5B8C7-1](https://user-images.githubusercontent.com/4013224/188283562-99f67a03-a7b4-415d-9bf6-87d1ebff3f10.jpeg)

**Comment by nrzky** [comment]
I mostly look at the logs from my mobile device. It will be very delicious.

**Reply by Grezor** [reply]
To see to display some very important information in the mobile version.
